CMakeLists.txt

```diff
@@ -86,6 +94,10 @@ else()
   add_definitions(-DNO_VULKAN)
 endif()
 
+if(ODROID)
+  set(VULKAN OFF)
```

Comment:
Eventually we may want to remove this define.  If it's a matter of an undefined reference or something, maybe we can just ifdef something more specific.  I know people on Raspberry Pi have had issues too, so I'd rather not laser-focus on ODROID only.

Is it just errors like this?
```
VulkanImage.cpp:89:11: error: cannot convert ‘std::nullptr_t’ to ‘VkImage {aka long long unsigned int}’ in assignment
    image_ = nullptr;
```

If so, these are definitely fixable.  That error is fixed, but in that case the correct code would be `image_ = VK_NULL_HANDLE;`.  It's just a bug in PPSSPP, and it should be fixed.

-[Unknown]

Reply:
I get linkage errors if I let pass it to ``add_definitions(-DNO_VULKAN)``

Reply:
Which linkage errors?  The only Vulkan code we link with is in this folder:
https://github.com/hrydgard/ppsspp/tree/master/ext/vulkan

So in theory it's all fixable inside PPSSPP's source code.  We're not linking with any system code AFAIK.

-[Unknown]

Reply:
I will get you postet, rebuilding it atm.

Reply:
Here it is https://pastebin.com/TQkDwybX

Reply:
Hm.  That means that CommonVulkan or at least VulkanMemory.cpp was not linked.  It should still be linked, even though it won't be used.  I don't see any conditionals preventing it from being included in current master:

https://github.com/hrydgard/ppsspp/blob/f87dcefdd89b72953c6b7172ac3e887d9a7f823b/CMakeLists.txt#L400-L410

-[Unknown]

Reply:
Oh, maybe it's some sort of link order problem?  But Common is before native:

https://github.com/hrydgard/ppsspp/blob/f87dcefdd89b72953c6b7172ac3e887d9a7f823b/CMakeLists.txt#L1748

-[Unknown]

Reply:
maybe it's an ccache problem, I will rebuild it but not know it has to wait a little bit